src/site/stages/build/process-cms-exports/schemas/transformed/paragraph-list-of-link-teasers.js

```diff
@@ -0,0 +1,12 @@
+const contentModelType = 'paragraph-list_of_link_teasers';
+
+module.exports = {
+  $id: contentModelType,
+  type: 'object',
+  baseType: {
+    enum: ['paragraph'],
```

Comment:
Is `properties` intentionally left out as a wrapper for `baseType` and `contentModelType`?

Reply:
So these differ only a little for now because they're incomplete. I just needed something to refer to from the `node-page.js` schema. Once we get around to filling them in, we can decide whether it makes sense to pull some common elements together in one place.

Reply:
Gotcha :+1: